game/src/test/scala/io/aigar/controller/GameControllerSpec.scala

```diff
@@ -8,15 +8,25 @@ import org.json4s.JsonDSL._
 
 import org.scalatra.test.specs2._
 import org.specs2.matcher._
+import org.specs2.specification.BeforeAfterEach
 
-class GameControlleSpec extends MutableScalatraSpec with JsonMatchers {
+class GameControllerSpec extends MutableScalatraSpec
+    with JsonMatchers
+    with BeforeAfterEach {
   implicit val jsonFormats: Formats = DefaultFormats
+  sequential
 
   val game = new GameThread
   game.updateGames // run once to initialize the game states
-
   addServlet(new GameController(game), "/*")
 
+  def cleanGame = {
+    game.actionQueue.clear()
+  }
+
+  def before = cleanGame
+  def after = cleanGame
+
```

Comment:
Smart choice to do it twice.. I wouldn't have thought of doing it... Maybe this is why my tests are failing with DB! I will try to clear my database before and after.